collections/modis.yaml

```diff
@@ -29,6 +29,9 @@ Tags:
   - natural resource
   - disaster response
   - open data
+  - terra
+  - aqua
+  - NASA
```

Comment:
Should "modis" also be added to the tags?

Reply:
Good idea, added.